src/index.js

```diff
@@ -29,9 +29,12 @@ const args = config.cliArgs;
 app.use(bodyParser.json({limit: args.limit}));
 app.use(bodyParser.urlencoded({extended: true, limit: args.limit}));
 
-const accessLogStream = fs.createWriteStream(args.logdir + '/access.log', {
-  flags: 'a'
-});
+const accessLogStream = fs.createWriteStream(
+  config.cliArgs.logdir + '/access.log',
```

Comment:
Variable `args` is already an alias for `config.cliArgs` used throughout the whole file, we should probably keep it for consistency.